LEGO1/beachhouseentity.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BEACHHOUSEENTITY_H
+#define BEACHHOUSEENTITY_H
+
+#include "buildingentity.h"
+
+#ifndef undefined4
+#define undefined4 int
+#endif

```

Comment:
Would be useful to put this in a global header `unknown_types.h`, to be used globally.
Grepping for types in this header allows us to find types that are still todo.